src/fragments/forms/map-form/components/place-and-directions/components/route-details/i18n/route-details.i18n.cs-cz.js

```diff
@@ -0,0 +1,24 @@
+
+export default {
+  routeDetails: {
+    routeDetails: 'Podrobnosti o trase',
+    warnings: 'Výstrahy',
+    routes: 'Trasy',
+    route: 'Trasa',
+    segments: 'Segmenty',
+    segment: 'Segment',
+    instructions: 'Instrukce',
+    instruction: 'Instrukce',
```

Comment:
This is the same word for the plural and the singular. Is this correct in czech?

Reply:
great, thanks :)